src/pages/docs/en/quickstart-guides/python.md

```diff
@@ -62,16 +58,30 @@ def hello_world():
     return "Hello, Space!"
 ```
 
-
 ## Step 2: Run it on Space
+
 Make sure you have the Space CLI installed & authenticated, then run the following command:
 
 ```bash
 space new
 ```
-This will create a new Space project in the current directory. You can then run the following command to push your code to Space:
-> __Important__: before `space push`, we recommend creating a [`.spaceignore`](/docs/en/basics/revisions#ignoring-files-and-directories) file to ignore files that you don't want to be deployed to Space. For example, you should ignore `.git` and `.venv` folders.
 
+This will create a new Space project in the current directory.
+
+Edit the generated `Spacefile` to add a `run` command for your app:
+
+```yaml
+v: 0
+micros:
+  - name: python-app
+    src: .
+    engine: python3.9
+    run: uvicorn main:app
+```
+
+> ⚠️ Make sure that your Micro is configured to listen on the port number specified in the environment variable `PORT`.
```

Comment:
@pomdtr this might be confusing as it makes it seem like you need to configure something additional whereas if you use one of the supported frameworks you don't have to do anything